src/list.tsx

```diff
@@ -350,8 +351,12 @@ const RowContent = React.forwardRef<TouchableElement, RowContentProps>((props, r
         role,
         extra,
         dataAttributes,
+        'aria-label': ariaLabel,
     } = props;
 
+    // iOS voiceover reads links with multiple lines as separate links. By setting aria-label and marking content as aria-hidden, we can make it read the whole row as one link.
+    const iosVoiceOverLabelForLink = [title, subtitle, description, detail].filter(Boolean).join(' ');
```

Comment:
caveat: headline is not read because it isn't included here. The problem is that `headline` prop is a `React.Node`, so I can't extract the text

Reply:
Have you tried something like this? https://stackoverflow.com/questions/34204975/react-is-there-something-similar-to-node-textcontent

I know it may be a bit "hacky", but usually the headline prop is used for rendering very simple elements, so I would give it a try